bitcoin/src/pow.rs

```diff
@@ -221,10 +221,17 @@ impl Target {
     /// integer but `difficulty()` returns only 128 bits this means for targets below approximately
     /// `0xffff_ffff_ffff_ffff_ffff_ffff` `difficulty()` will saturate at `u128::MAX`.
     ///
+    /// # Panics
+    ///
+    /// Panics if `self` is zero (divide by zero).
```

Comment:
Should say that this is debug-assertions-only.

Reply:
I was under the impression div by zero panics, we just add the debug assertion so in debug mode it panics in a different spot (earlier).

Reply:
In that case I'd use non-debug assertion. It's quite likely compiler optimization will be able to remove the innermost one.

Reply:
Used as suggested.